source/nodejs/adaptivecards/src/schema.ts

```diff
@@ -134,7 +133,7 @@ export type TextColor = "default" | "dark" | "light" | "accent" | "good" | "warn
 export type TextInputStyle = "text" | "tel" | "url" | "email";
 export type TextSize = "small" | "normal" | "medium" | "large" | "extraLarge";
 export type TextWeight = "lighter" | "normal" | "bolder";
-export type Separation = "none" | "default" | "strong";
+export type Spacing = "none" | "default" | "small" | "medium" | "large" | "extraLarge";
```

Comment:
All these enums are already declared in Enums, why do we need to re-declare them here, and in a different way for that matter? Let's use the real enums instead of duplicating. That applies to Spacing and all the others.

Reply:
And remove the locally defined enums from schema.ts

Reply:
Fixed